test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -234,7 +234,7 @@ private V1beta1RoleBinding getPravegaOperatorRoleBinding() {
 
     private V1Deployment getPravegaOperatorDeployment() {
         V1Container container = new V1ContainerBuilder().withName(PRAVEGA_OPERATOR)
-                                                        .withImage(DOCKER_REGISTRY + PREFIX + "/pravega-operator:" + PRAVEGA_OPERATOR_VERSION)
+                                                        .withImage("pravega/pravega-operator:" + PRAVEGA_OPERATOR_VERSION)
```

Comment:
Why we do not follow the same pattern with the operator as with Pravega/Bookkeeper? Is the operator image being published differently?

Reply:
Yes, as of now there are no jenkins jobs which pushes snapshot pravega-operator and zookeeper-operator to a private docker repository. 
We can modify it to have a same pattern once the operator snapshot docker images are published similar to Pravega/ Bookkeeper. (As of now, operator images are always fetched from dockerhub.)